ui/yarn.lock

```diff
@@ -0,0 +1,4 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
I suspect you ran `yarn start` from the wrong folder and this got created (I've been there, and done that 😄 ). Can you remove this file please?

Reply:
Yep, did see that. good catch.